doc/apiv3-documentation.apib

```diff
@@ -4326,6 +4723,21 @@ For more details and all possible responses see the general specification of [Fo
 
     [Example Form][]
 
+## Relations [/api/v3/work_packages/{work_package_id}/relations]
+
+## List relations [GET]
+
+Lists all relations this work package is involved in.
+
++ Parameters
+    + work_package_id (required, integer, `1`) ... Work package id
+
++ Response 302 (text/plain)
+
+  + Body
+
+            You are being redirected to /api/v3/relations?involved={work_package_id}
```

Comment:
Fair response, but why add the end point in the first place and link to it in the work package resource when all it does is redirecting.